src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheckTest.java

```diff
@@ -686,7 +686,9 @@ public void testMultiplePatternMatchesSecondPatternIsLonger() throws Exception {
         checkConfig.addAttribute("standardPackageRegExp", "junit");
 
         createChecker(checkConfig);
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        final String[] expected = {
+            "4: " + getCheckMessage(MSG_NONGROUP_EXPECTED, STD, "org.junit.Test"),
```

Comment:
Please move case to brand new test to not conflict with existing tests.

Reply:
Ok, I've reinstated the original tests and added new tests with a separate input file

Reply:
i see we have 3 new tests now when I was only expecting 1. Is all of them needed for the pitest lines?

Reply:
You're right, on further testing I found that not all the tests are necessary. Originally I just made a copy of the 3 modified tests, but the first and third don't actually contribute to killing the pitest mutations.

I'll remove the 2 redundant tests.

Reply:
Thanks.